tests/export/html/test_markup_compatibility.py

```diff
@@ -91,3 +91,71 @@ def test_textbox_with_content_outside_of_textbox(self):
             </p>
         '''
         self.assert_document_generates_html(document, expected_html)
+
+    def test_fallback_is_only_text(self):
+        document_xml = '''
+            <p>
+                <r><t>AAA</t></r>
+                <r>
+                    <t>BBB</t>
+                    <AlternateContent>
+                        <Fallback>
+                            <r>
```

Comment:
My understanding was that the fallback would need to only contain those nodes that could otherwise be contained by the grandparent. In this case, you have a fallback with a run. The grandparent is a run. So you have nested runs, which isn't valid. In this case, assuming my understanding is correct (which it may not be), what you really want to test is `<Fallback><t>CCC</t></Fallback>`.